Classes/Netlogix/JsonApiOrg/Property/TypeConverter/Entity/PersistentObjectFromTopLevelArrayConverter.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace Netlogix\JsonApiOrg\Property\TypeConverter\Entity;
+
+use Neos\Flow\Annotations as Flow;
+use Neos\Flow\Property\Exception as PropertyException;
+use Neos\Flow\Property\PropertyMapper;
+use Neos\Flow\Property\PropertyMappingConfigurationInterface;
+
+class PersistentObjectFromTopLevelArrayConverter extends PersistentObjectConverter
+{
+    const TARGET_TYPE = '__target_type';
+
+    protected $priority = 1565610123;
+
+    protected $sourceTypes = ['array'];
+
+    /**
+     * @Flow\Inject
+     * @var PropertyMapper
+     */
+    protected $propertyMapper;
+
+    /**
+     * Only convert non-persistent types
+     *
+     * @param mixed $source
+     * @param string $targetType
+     * @return boolean
+     */
+    public function canConvertFrom($source, $targetType)
+    {
+        if (!is_array($source)) {
+            return false;
+        }
+        if (!array_key_exists('data', $source)) {
```

Comment:
Das würde ich auch in die condition oben drüber mit einbauen

Reply:
Done.